packages/lodestar-params/src/beaconParams.ts

```diff
@@ -0,0 +1,86 @@
+import {IBeaconParams} from "./interface";
+import {
+  ContainerType, BigIntUintType, NumberUintType, BitVectorType, ByteVectorType,
+} from "@chainsafe/ssz";
+
+const Number64 = new NumberUintType({byteLength: 8});
+const BitVector8 = new BitVectorType({length: 8});
+const ByteVector32 = new ByteVectorType({length: 4});
```

Comment:
ByteVector32 should be length 32

This number is different than in number types.
We usually discuss numbers in bitlength (eg: uint64 is 64 bits, 8 bytes)
Bytevectors usually discussed in bytelength (eg: bytes32 is 32 bytes)